Gopkg.toml

```diff
@@ -19,6 +19,10 @@
   name = "github.com/docker/cli"
   revision = "f95ca8e1ba6c22c9abcdbf65e8dcc39c53958bba"
 
+[[constraint]]
```

Comment:
Out of curiosity, why do you need this constraint, as it seems it's no-op (no vendoring update or no Gopkg.lock)? 🤔

Reply:
Ah, good call; removed.  The package was already included.